services/stats/stats.go

```diff
@@ -210,6 +211,23 @@ func (s *HandleT) NewSampledTaggedStat(Name, StatType string, tags Tags) (rStats
 	return newTaggedStat(Name, StatType, tags, statsSamplingRate)
 }
 
+func CleanupTagsBasedOnDeploymentType(tags Tags, key string) Tags {
```

Comment:
We are passing the `key` but not using it?

Reply:
fixed.